### ISSUE TITLE
Multi-target detection and classification

### ISSUE THREAD

**Issue by GGOBOND** [xairc/lung_nodule_detector#10]
Hi, 
can this code be changed from single target detection to multi-target detection and classification?  Because there are other diseases besides pulmonary nodules.

**Comment by dypark86** [comment]
yes, you can change code for multi target detection or classification.
in general object detection split with
object recognition and object classification.

this code is just for nodule(object) recognition.
you can change this code for multi target with faster r-cnn architecture.
